vita3k/config/include/config/config.h

```diff
@@ -17,9 +17,39 @@
 
 #pragma once
 
-#include <psp2/system_param.h>
 #include <spdlog/spdlog.h>
 
+// temporary enum copy, should be moved to some header like util/types.h before merge
```

Comment:
i confirm, here is wrong haha, and this relate apputil, go just use like rpcs3 ans set in inside include corresponding :)